.github/workflows/tests.yml

```diff
@@ -87,6 +87,11 @@ jobs:
         source '/usr/share/miniconda/etc/profile.d/conda.sh'
         conda activate improver_${{ matrix.env }}
         black --check .
+    - name: flake8
+      run: |
+        source '/usr/share/miniconda/etc/profile.d/conda.sh'
+        conda activate improver_${{ matrix.env }}
+        flake8 improver improver_tests
```

Comment:
Suggest adding `|| true` so that github actions runs flake8, but doesn't fail the CI job yet.
Once the issues identified by flake8 are fixed (in a separate PR) that bypass can be removed and enforcement can start.

Reply:
Done.